src/main/java/frc/trigon/robot/subsystems/arm/ArmConstants.java

```diff
@@ -0,0 +1,100 @@
+package frc.trigon.robot.subsystems.arm;
+
+import com.ctre.phoenix6.hardware.CANcoder;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public class ArmConstants {
+    // 2 motors for angle - CANSparkMax, Neo (Brushless)
+    // 2 motors for elevator/opening - CANSparkMax, Neo (Brushless)
+    // CANcoder - angle encoder
+    // TalonSRX, mag encoder - elevator encoder
+    // PIDController
+    // Enum arm state (angle, elevatorPosition)
+
+    // configs:
+    // encoders:
+    // direction / phase
+    // offset
+    // (only cancoder) range
+    private static final int
+            ANGLE_MOTOR_MASTER_ID = 0,
+            ANGLE_MOTOR_FOLLOWER_ID = 1,
+            ELEVATOR_MOTOR_MASTER_ID = 0,
+            ELEVATOR_MOTOR_FOLLOWER_ID = 1,
+            ANGLE_ENCODER_ID = 0,
+            ELEVATOR_ENCODER_ID = 1;
+    private static final boolean
+            ANGLE_MOTOR_MASTER_INVERTED = false,
+            ANGLE_MOTOR_FOLLOWER_INVERTED = false,
+            ELEVATOR_MOTOR_MASTER_INVERTED = false,
+            ELEVATOR_MOTOR_FOLLOWER_INVERTED = false;
+    private static final CANSparkMax.IdleMode
+            ANGLE_MOTORS_IDLE_MODE = CANSparkMax.IdleMode.kBrake,
+            ELEVATOR_MOTORS_IDLE_MODE = CANSparkMax.IdleMode.kBrake;
+    private static final int VOLTAGE_COMPENSATION_SATURATION = 12;
+    private static final int
+            P = 0,
+            I = 0,
+            D = 0;
```

Comment:
Move this after where you define the motors, and put it next to the pid controller